github.user.styl

```diff
@@ -974,20 +974,20 @@ for c, i in C {
             &:checked {
                 z = '<circle cx="256" cy="256" r="128" fill="white"/>'
                 background-image: s("url('%s%s</svg>') %s", r(prefix), r(z), i)
-                c: 0 ac2 ac3
+                c: 0 ac3 ac5
             }
             &:focus { border-radius: 50% }
-            &:hover { c: 0 ac1 ac3 }
+            &:hover { c: 0 ac1 bg2 }
         }
 
         &[type='checkbox'] {
             border-radius: 2px
             &:checked {
                 z ='<path fill="white" d="M99 213l-1 89 104 102 205-201v-91L203 315 99 213z"/>'
                 background-image: s("url('%s%s</svg>') %s", r(prefix), r(z), i)
-                c: 0 ac3 ac4
+                c: 0 ac3 ac5
```

Comment:
this chnage is due to the fact checkboxes can appear on colored bg, if this color is too bright it may tend to be less contrast, but if you prefer it its OK too.

In any case both the radio/checkbox checked states should match also.